File: python_ToSXs/python/02-caudieukien/28.py
BONUS_LEVEL_ONE = 0.05
BONUS_LEVEL_TWO = 0.1
BONUS_LEVEL_THREE = 0.2

totalSale = 90
incentive = BONUS_LEVEL_THREE

if totalSale <= 100:
    incentive = BONUS_LEVEL_ONE
elif totalSale <= 300:
    incentive = BONUS_LEVEL_TWO

commission = totalSale * incentive
print(f"Hoa hồng: {commission}")

File: cpp_e9zMZ/cpp/02-caudieukien/28.cpp
#include <iostream>
using namespace std;

int main() {
    int totalSale = 1800;
    const float BONUS_LEVEL_ONE = 0.05;
    const float BONUS_LEVEL_TWO = 0.1;
    const float BONUS_LEVEL_THREE = 0.2;
    float incentive = BONUS_LEVEL_THREE;

    if (totalSale <= 100) {
        incentive = BONUS_LEVEL_ONE;
    } else if (totalSale <= 300) {
        incentive = BONUS_LEVEL_TWO;
    }

    float salary = totalSale * incentive;
    cout << "Hoa hồng: " << salary;
}